.env.production.sample

```diff
@@ -214,3 +214,10 @@ STREAMING_CLUSTER_NUM=1
 # SAML_UID_ATTRIBUTE="urn:oid:0.9.2342.19200300.100.1.1"
 # SAML_ATTRIBUTES_STATEMENTS_VERIFIED=
 # SAML_ATTRIBUTES_STATEMENTS_VERIFIED_EMAIL=
+
+# Use HTTP proxy for outgoing request (optional)
+# http_proxy=http://gateway.local:8118
+# Access control for darknet sites.
+# ALLOW_ACCESS_TO_DARKNET=true
```

Comment:
Is there any reason to say "darknet" instead of "tor"? Does this setup support more than just Tor? There are negative connotations with "darknet" so I would prefer another term used for these variables and method names.

Reply:
"Darknet" includes not only Tor but also I2P.
I also feel that this name is not good. Should I change it to "SKIP_PRIVATE_ADDRESS_CHECK"?

Reply:
HIDDEN_SERVICE would be an okay alternative wherever DARKNET is used currently

Reply:
it's nice.

Reply:
for the record in pleroma the feature is called "upstream proxy support" and has no references to "darknet" or "hidden service" since that is what the feature really is at its core.

Reply:
"Hidden service" is a former name for that functionality. Tor has switched to calling those Onion services: https://www.torproject.org/docs/onion-services.html.en